.github/workflows/reusable-build.yml

```diff
@@ -161,15 +161,6 @@ jobs:
         with:
           containers: akmod:coreos-${{ env.fedora_version }}
 
-      - name: Verify Chainguard images
-        if: matrix.base_name != 'bluefin' && matrix.base_name != 'aurora'
```

Comment:
We do this for provenance of the images we pull binaries from

Reply:
We are no longer installing the dev tools from chainguards images. The only thing we install not from a repo is kind now.

Everything else is coming from Brew

Reply:
Awesome! Just wanted to be sure